Sources/SourceKitLSP/Swift/SwiftCommand.swift

```diff
@@ -15,15 +15,20 @@ import LanguageServerProtocol
 ///
 /// All commands from the Swift LSP should be listed here.
 public let builtinSwiftCommands: [String] = [
-  SemanticRefactorCommand.self
-].map { $0.identifier }
+  SemanticRefactorCommand.self,
+  ExpandMacroCommand.self,
+].map { (command: any SwiftCommand.Type) in
+  command.identifier
+}
 
 /// A `Command` that should be executed by Swift's language server.
 public protocol SwiftCommand: Codable, Hashable, LSPAnyCodable {
   static var identifier: String { get }
   var title: String { get set }
 }
 
+public typealias SwiftCommandHandler = (any SwiftCommand) async throws -> LSPAny
+
```

Comment:
I think we only use this once so I’d just use the function type inline. Just means that you don’t need to look up what `SwiftCommandHandler`. Just means that you don’t need to look up what `SwiftCommandHandler ` is when reading the definition of `commandsAndHandlers`.